tests/vmi_configuration_test.go

```diff
@@ -1065,7 +1089,10 @@ var _ = Describe("[sig-compute]Configurations", func() {
 				return false
 			}
 			BeforeEach(func() {
-				hugepagesVmi = tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
+				hugepagesVmi = libvmi.NewCirros(
+					libvmi.WithNetwork(v1.DefaultPodNetwork()),
+					libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
```

Comment:
Is this explicitly needed?

Reply:
Removed masquerade interfaces where not needed: https://github.com/kubevirt/kubevirt/pull/7545/commits/70d5a9006b983fb4af14d29bd2ba6c2269bedb46